sql/init.sql

```diff
@@ -6,6 +6,11 @@ DROP TABLE IF EXISTS Items;
 DROP TABLE IF EXISTS Orders;
 SET FOREIGN_KEY_CHECKS = 1;
 
+-- Set increment settings --
+
+SET auto_increment_increment = 1;
+SET auto_increment_offset = 1;
```

Comment:
Do we need to keep the increment settings or will the database reset to the weird inc 10 and offset 4?